ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -1726,6 +1726,40 @@ def test_torch_instance_aminmax(
     )
 
 
+# bernoulli
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bernoulli",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
Valid returns all dtypes supported by the function.

Reply:
sorry, I missed to changed it...😅 Thanks for pointing out. I will commit your suggestion